### DISCUSSION TITLE
Collapsable Aside

### DISCUSSION THREAD

**Discussion by Darkflame72** [withastro/starlight#1693] [Feature Requests]
### What version of `starlight` are you using?

0.21.0

### What is your idea?

Collapsable aside for details or expansion info that should not be visible by default. This would be similar to the [details container](https://vitepress.dev/guide/markdown#custom-containers) from VitePress.

### Why is this feature necessary?

Often with complex documentation you want notes or expansion details that aren't needed to be displayed by default but can provide extra information to those interested. This would be a collapsed aside by default with the option to expand it to provide the information if the reader is interested.

### Do you have examples of this feature in other projects?

- https://vitepress.dev/guide/markdown#custom-containers

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Just adding a reference to this [`<Details>` component feature request](https://github.com/withastro/starlight/discussions/1491#discussioncomment-8889999) that may be highly related to this one.

**Reply by Darkflame72** [reply]
Can't believe I missed it, will close this in favour of that.

**Comment by Darkflame72** [comment]
Duplicate of https://github.com/withastro/starlight/discussions/1491#discussioncomment-8889999
